ArchiSteamFarm/Core/Utilities.cs

```diff
@@ -319,6 +325,42 @@ public static class Utilities {
 			}
 		}
 
+		internal static bool IsWeakPassword(string password, ISet<string>? additionallyForbiddenPhrases = null) {
+			if (string.IsNullOrEmpty(password)) {
+				throw new ArgumentNullException(nameof(password));
+			}
+
+			HashSet<string> forbiddenPhrases = ForbiddenPasswordPhrases.ToHashSet();
+
+			if (additionallyForbiddenPhrases?.Any() == true) {
```

Comment:
You have `ISet`, make use of it with `set?.Count > 0`. Also, `UnionWith()` can handle empty collections, so just `!= null` will be enough.